source/cloud-security/office365/monitoring-office365-activity.rst

```diff
@@ -413,30 +413,10 @@ For example, the following configuration pulls only the ``Audit.AzureActiveDirec
 
 Replace ``<YOUR_TENANT_ID>``, ``<YOUR_CLIENT_ID>``, and ``<YOUR_CLIENT_SECRET>`` with the organization's credentials for the tenant.
 
-Enabling dashboard visualization for the Office 365 module
+Visualizing the Office 365 activity
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
**nitpick**: the underline size should be the same count of characters of the title. I think this emits a warning in the building time.